bot_modules/http_server.js

```diff
@@ -55,6 +56,7 @@ module.exports = {
 						return;
 					}
 					// check for and run command
+					console.log('run command' + data.command);
```

Comment:
no space between `command` and the command name? how bout some backticks

Reply:
squashed into commit